RefreshTests.GameServer/GameServer/TestRefreshGameServer.cs

```diff
@@ -64,7 +68,17 @@ protected override void SetupServices()
         this._server.AddService<TimeProviderService>(this.DateTimeProvider);
         this._server.AddService<CategoryService>();
         this._server.AddService<MatchService>();
-        this._server.AddService<CommandService>();
         this._server.AddService<ImportService>();
+        this._server.AddService<LevelListOverrideService>();
+        this._server.AddService<CommandService>();
+    }
+    
+    [Pure]
+    public TService GetService<TService>() where TService : Service
+    {
+        List<Service> services = (List<Service>)typeof(BunkumServer).GetField("_services", BindingFlags.Instance | BindingFlags.NonPublic)!
+            .GetValue(this._server)!;
+
+        return (TService)services.First(s => typeof(TService) == s.GetType());
     }
```

Comment:
Im not sure how i feel about this, this should be something exposed in Bunkum, but since its non-critical test code it should be fine, non-blocking for merge but i'd like to see this more cleanly resolved at some point

Reply:
Yeah, the reasoning behind that behind choice in Bunkum was I wanted to discourage handling of Bunkum's objects, but as Bunkum grew that use-case became more acceptable (think database providers outside of the main BunkumServer for instance). I'll definitely implement this in Bunkum.